src/hii_water.py

```diff
@@ -5,24 +5,29 @@
 
 
 class HIIWater(HIITask):
-    ee_rootdir = "projects/HII/v1"
-    ee_driverdir = "driver/water"
-    # TODO: figure out inputs that will be updated
+    scale = 300
+    gpw_cadence = 5
+    wide_river_width = 30  # meters
+    coastal_settlements_dist = 4000  # meters
+    indirect_distance = 15000  # meters
+    DECAY_CONSTANT = -0.0002
```

Comment:
Doesn't really matter but generally the convention is "constants" (which these aren't really, in python, but they're marked as such by upper case) before variables.

Reply:
Sorry, just meant to change the order, not capitalize these... `scale` is actually a variable property from `task_base` so you want to use it, lowercase.